app.py

```diff
@@ -20,11 +20,12 @@ def placeUser(latitude, longitude):
     except KeyboardInterrupt:
         log.info("GOODBYE")
         sys.exit(1)
-    except Exception, err:
+    except Exception as err:
         from app.util import log
         log.exception("Unknown exception")
         abort(500)
```

Comment:
We're still seeing these inconsistencies. I've started just catching them and logging them like so:

```python
except Exception:
     log.exception("Uncaught exception")
```

so we don't even need the `err` in scope.